src/Symfony/Component/DependencyInjection/Argument/AbstractArgument.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Argument;
+
+/**
+ * Represents an abstract parameter, which have to be replaced by one of CompilerPasses.
+ */
+final class AbstractArgument
+{
+    /** @var string */
+    private $serviceId;
+
+    /** @var string */
+    private $argKey;
+
+    /** @var string */
```

Comment:
please remove these annotations, the constructor already mentions their type

Reply:
fixed